.github/release-initial-checklist.md

```diff
@@ -40,7 +40,11 @@ Additionally, make sure to differentiate between things in the testing notes tha
 ## Update Pull Request description and get approvals
 
 * [ ] Go through the description of the release pull request and edit it to update all the sections and checklist instructions there.
-* [ ] Ask a team member to review the changes in the release pull request and for anyone who has done testing that they approve the pull request.
+* [ ] Ask the porter of Rubik and Kirigami to review the changes in the release pull request and to approve the PR if everything looks good. The porter of each team is responsible for testing the PRs created by members of their own team. That means Rubik porter will mostly test Cart & Checkout changes while Kirigami porter will test other blocks.
+  * If all PRs are testing as expected, continue with the release.
+  * If one or more PRs are not testing as expected: ping the PR authors and the porter of the relevant team and ask them if the change is a release blocker or not (you can also ping the team lead if any of them is not available). In general, if it's not a regression or there is no product/marketing reason why that PR is a blocker, all other PRs should default to not being blockers.
+    * If there are blockers: stop the release and ask the PR author and team porter to fix them.
+    * If some PRs are not testing as expected but they are not blockers: remove the from testing steps and changelog, open an issue (or reopen the original one) and proceed with the release.
```

Comment:
Same comment, I think we need to mention here that we need to revert the PR's commits